frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -3,17 +3,28 @@ import classNames from 'classnames';
 
 import { LoadingInline } from '../../utils';
 
-export const DetailItem: React.FC<DetailItemProps> = React.memo(({ title, value, isLoading }) => (
-  <React.Fragment>
-    <dt className="co-details-card__item-title">{title}</dt>
-    <dd className={classNames('co-details-card__item-value', {'text-secondary': !value})}>
-      {isLoading ? <LoadingInline /> : value || 'Unavailable'}
-    </dd>
-  </React.Fragment>
-));
+export const DetailItem: React.FC<DetailItemProps> = React.memo(
+  ({ title, isLoading, children, error = false }) => {
+    let status: React.ReactNode|string;
+    if (isLoading) {
+      status = <LoadingInline />;
+    } else if (error) {
+      status = 'Unavailable';
```

Comment:
if you do 
```suggestion
      status = <span className="text-secondary">Unavailable</span>;
```

then the status type can be just `React.ReactNode` and later we dont have to use classNames for `dd` element

Reply:
ack